src/EventListener/MapperListener.php

```diff
@@ -130,16 +91,22 @@ private function getViewAnnotationByController($controller): ?Dto\View
             }
         }
 
+        /** @var Dto\View $attributes */
+        $attributes = array_map(
+            static function (ReflectionAttribute $attribute): object { return $attribute->newInstance(); },
```

Comment:
Same here with arrow functions :)

Reply:
Implemented in https://github.com/byWulf/apitk-dtomapper-bundle/pull/15/commits/a4298e9508dde0ad8f589c100ef0b575653edc96